modules/network-observability-pktdrop-overview.adoc

```diff
@@ -13,13 +13,12 @@ You can configure graphical representation of network flow records with packet l
 
 * Performance optimization: With a clearer picture of packet drops, you can take steps to optimize network performance, such as adjust buffer sizes, reconfigure routing paths, or implement Quality of Service (QoS) measures.
 
-When packet drop tracking is enabled, you can see the following metrics represented in a chart in the *Overview*.
+When packet drop tracking is enabled, you can see the following panels in the *Overview* by default with more are available to add in *Manage panels*:
 
-* Top X flow dropped rates stacked
-* Total dropped rate
-* Top X dropped state
-* Top X dropped cause
-* Top X flow dropped rates stacked with total
+* Top X packet dropped state stacked with total
+* Top X packet dropped cause stacked with total
+* Top X average dropped packets rates 
+* Top X dropped packets rates stacked with total
```

Comment:
@Amoghrd Here I added the packet drop panels, and I say these are the defaults that are shown and more are available in *Manage panels*.

Reply:
but you havent listed the non-default panels like you have done in DNS docs